test/win32.js

```diff
@@ -90,6 +90,21 @@ function setCopyrightAndCompanyNameTest (appCopyright, companyName) {
   )
 }
 
+function setRequestedExecutionLevelTest (requestedExecutionLevel) {
+  var opts = {
+    win32metadata: {
+      requestedExecutionLevel: 'asInvoker'
```

Comment:
I've decided to remove the default, and by default don't modify the assembly.   Whatever upstream wants to put as the default in the electron.exe should probably stay as the default.